docs/source/loading_datasets.rst

```diff
@@ -441,6 +441,29 @@ You can even set the environment variable `HF_DATASETS_OFFLINE` to ``1`` to tell
 This mode disables all the network calls of the library.
 This way, instead of waiting for a dataset builder download to time out, the library looks directly at the cache.
 
+.. _load_dataset_load_builder:
```

Comment:
.. load_dataset_builder?

Reply:
`load_dataset` is the prefix of all the anchors of this page of the documentation x)